src/index.html

```diff
@@ -7,7 +7,47 @@
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body >
+    <main class="card" data-qa="card">
```

Comment:
```suggestion
    <article class="card" data-qa="card">
```

Reply:
getting an error after adding spaces and 'article' , didn't work on stars yet .  Element not found for capturing: [data-qa="card"]
x Close Browser
Puppeteer encountered an error while running scenario "Link with data-qa_hover"
TimeoutError: Waiting for selector `[data-qa="hover"]` failed: Waiting failed: 30000ms exceeded
      COMMAND | Executing core for "report"